geozero/src/chaining.rs

```diff
@@ -0,0 +1,189 @@
+//! Chaining and duplexing processors.
+
+use crate::error::Result;
+use crate::events::GeomEventProcessor;
+use crate::events::*;
+
+// ------- Chain events -------
+
+/// Processing geometry events and passing events to a chained visitor
+pub trait CĥainedGeomEventProcessor {
```

Comment:
Oh wow, programming with diacritics! Exciting! 😉

Reply:
No idea, how I did that :rofl: